Modules/xASL_module_Population.m

```diff
@@ -30,11 +30,11 @@
 % EXAMPLE: [~, x] = xASL_module_Population(x);
 % -----------------------------------------------------------------------------------------------------------------------------------------------------
 % Copyright 2015-2024 ExploreASL
-% Licensed under Apache 2.0, see permissions and limitations at
-% https://github.com/ExploreASL/ExploreASL/blob/main/LICENSE
-% you may only use this file in compliance with the License.
-% __________________________________
-
+% Licensed under Apache 2.0, see permissions and limitations at

```

Comment:
I see you had the same problem. Probably Matlab defaulting to different line endings in the m-files than how I programmed it with the cell splitting and joining thingy

Reply:
could be. lets ignore this as a minor nuissance...